app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -53,4 +60,37 @@ public String getDatabaseUserdataPath() {
         return this.androidFilesDir + Constants.DB_USERDATA_PATH;
     }
 
+    /**
+     * Serialize the created proxy instead of this instance.
+     */
+    private Object writeReplace() {
+        return new SerializationProxy(this);
+    }
+
+    /**
+     * Ensure that no instance of this class is created because it was present in the stream. A correct
+     * stream should only contain instances of the proxy.
+     */
+    private void readObject(final ObjectInputStream stream) throws InvalidObjectException {
+        throw new InvalidObjectException("Proxy required.");
+    }
+
+    /**
+     * Represents the logical state of this class and copies the data from that class without
+     * any consistency checking or defensive copying.
+     * Used for the Serialization Proxy Pattern.
+     */
+    @SuppressWarnings("PMD.AccessorClassGeneration")
```

Comment:
What is different about this serializable proxy that it needs this suppression (other serializable proxies don't have this suppression, e.g. `SimpleKey` of `RSAKey`)?